docs/xdr/features/investigate/archives.md

```diff
@@ -0,0 +1,80 @@
+
+!!! note 
+    To use the Archives feature, you must have the necessary permissions. For more information, contact our support. 
+
+!!! info 
+    You must subscribe to Amazon S3 or OVHCloud cloud storage features to be able to configure Archives. The pricing depends on the volume of data you want to archive. SEKOIA.IO support team can help you estimate the costs.
+
+## The Archives page
+
+The Archives feature is accessible on the `Archives page` in the Operations Center menu. 
+
+On this page, a table lists all archives created by the user. 
+
+| Column | Description |
+| --- | --- |
+| Name | Name of the archive |
+| Provider | Where the archive is stored |
+| Number of snapshots | How many snapshots were made since the archive was first created   |
+| Created at | Creation date |
+| Status  | Could be `success` or `error` |
+
+### Views from archive tab
+
+The archives tab lists all the different restorations that have been realized on this community. This view is only accessible internally for now.
+
+## Create an archive
+
+To create an archive, follow these instructions:
+
+=== "Amazon S3"
+
+    1. **Choose your archive destination**: it can be Amazon S3 storage or OVHcloud Object storage
+    2. **Configure your bucket** (the data associated with the storage space) by providing the following information
+        
+        | Name | Description |
+        | --- | --- |
+        | Container/Bucket | Name of the bucket |
+        | Region | The region associated with the bucket |
+        | Access Key ID | ? |
+        | Access Key Secret | ? |
+
+    3. **Describe your archive **
+    4. **Add GPK Key to access your events (optional)**: Archives are stored as GPG encrypted files so that the selected cloud provider does not have access to your events. You will always be able to load your archives inside SEKOIA.IO but if you want to access your events outside of the app, you can add your own public key.
+
+=== "OVHCloud"
+
+    1. **Choose your archive destination**: it can be Amazon S3 storage or OVHcloud Object storage
+    2. **Configure your bucket** (the data associated with the storage space) by providing the following information
+        
+        | Name | Description |
+        | --- | --- |
+        | User name |  |
+        | Password |  |
+        | Tenant name |  |
+        | Container name |  |
+
+    3. **Describe your archive**
+    4. **Add GPK Key to access your events (optional)**: Archives are stored as GPG encrypted files so that the selected cloud provider does not have access to your events. You will always be able to load your archives inside SEKOIA.IO but if you want to access your events outside of the app, you can add your own public key.
+
+!!! info
+    The format used in archives are logs compressed in JSON format and it keeps enrichments made by SEKOIA.IO. Since we keep the original logs and message fields, the volumes are therefore larger in the archives compared to the originals.
```

Comment:
```suggestion
    The format used in archives are logs compressed in JSON format with enrichments made by SEKOIA.IO. Since we keep the original logs and message fields, the volumes are therefore larger in the archives compared to the originals.
```